### PR TITLE
third try to fix gcloud auth

### PR DESCRIPTION
**Pull request by maxfisher-g**

### CODE REVIEW

File: .github/workflows/auth-test.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: 'Test gcloud authentication'
+
+on:
+  pull-request:
```
Comment (oliverchang) [inline .github/workflows/auth-test.yml:4]: Let's change this back to push now that we verified it's working.

File: .github/workflows/auth-test.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+
+    - name: Setup Environment
```
Comment (oliverchang) [inline .github/workflows/auth-test.yml:23]: I think this step is no longer needed as well.

File: .github/workflows/auth-test.yml (hunk 1 above)
Comment (maxfisher-g) [reply .github/workflows/auth-test.yml]: Deleted this check action

File: .github/workflows/auth-test.yml (hunk 2 above)
Comment (maxfisher-g) [reply .github/workflows/auth-test.yml]: It's needed if we use the exported credentials file (see [here](https://github.com/google-github-actions/auth#other-inputs under `create_credentials_file`)

Can't tell if it's used though

Comment (oliverchang) [reply .github/workflows/auth-test.yml]: `create_credentials_file` defaults to true, and it exports `GOOGLE_APPLICATION_CREDENTIALS` for us automatically right?

Comment (maxfisher-g) [reply .github/workflows/auth-test.yml]: Yes, provided the checkout action is used above

Comment (maxfisher-g) [reply .github/workflows/auth-test.yml]: Oh sorry, I forgot to push my latest change. I thought you were talking about the checkout action!

**Review by oliverchang — APPROVED** [review]

File: .github/workflows/auth-test.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+name: 'Test gcloud authentication'
```
[the remark this answers is not in the document]
Comment (maxfisher-g) [reply .github/workflows/auth-test.yml]: It's deleted

**Pull request merged**